pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -1,12 +1,12 @@
 #hero
-  - page_mode = PageMode.get(request)
-
   -# temporary for hoc2015: using id_s is a quick hack to only pull the hero desired.
   -# Generate a random set of hero images alternating between non-celeb and celeb.
   - heroes = DB[:cdo_homepage].where(id_s: "hero_#{page_mode}", type_s: "widehero", active_s: ["show","hide"], site_s: [request.site,"both"], language_supported_s: ["all", languageFilter]).all
 
-  - if page_mode == "minecraft"
+  - if page_mode == "feature-mc"
```

Comment:
Yes, the hero on homepage and banner on /learn will animate between the two. todo